src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -150,12 +150,15 @@ class CoreWorkerDirectTaskSubmitter {
   /// \param[in] addr The address of the worker.
   /// \param[in] task_queue_key The scheduling class of the worker.
   /// \param[in] was_error Whether the task failed to be submitted.
+  /// \param[in] error_detail The reason why it was errored.
```

Comment:
This file change is to add more detail when a worker is returned with was_error = true. I found it currently doesn't give any detail